File: BEE 1045.cpp
#include<bits/stdc++.h>

using namespace std;

int main(){
    double a, b, c;
    cin >> a >> b >> c;
    double arr[] = {a, b, c};
    sort(arr, arr+3);
    a = arr[2];
    b = arr[1];
    c = arr[0];
    if (a >= (b + c)){
        cout << "NAO FORMA TRIANGULO" << endl;
    }
    else if ((a * a) == ((b * b) + (c * c))){
        cout << "TRIANGULO RETANGULO" << endl;
    }
    else if ((a * a) > ((b * b) + (c * c))){
        cout << "TRIANGULO OBTUSANGULO" << endl;
    }
    else if ((a * a) < ((b * b) + (c * c))){
        cout << "TRIANGULO ACUTANGULO" << endl;
    }

    if (a == b && a == c){
        cout << "TRIANGULO EQUILATERO" << endl;
    }
    if ((a == b && a != c) || (b == c && b != a) || (a == c && a != b)){
        cout << "TRIANGULO ISOSCELES" << endl;
    }
    return 0;
}

File: BEE 1045.java
import java.io.IOException;
import java.util.Scanner;
import java.util.Arrays;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        double a = input.nextDouble();
        double b = input.nextDouble();
        double c = input.nextDouble();
        double[] arr = {a, b, c};
        Arrays.sort(arr);
        a = arr[2];
        b = arr[1];
        c = arr[0];
        if (a >= (b + c)){
            System.out.println("NAO FORMA TRIANGULO");
        }
        else if ((a * a) == ((b * b) + (c * c))){
            System.out.println("TRIANGULO RETANGULO");
        }
        else if ((a * a) > ((b * b) + (c * c))){
            System.out.println("TRIANGULO OBTUSANGULO");
        }
        else if ((a * a) < ((b * b) + (c * c))){
            System.out.println("TRIANGULO ACUTANGULO");
        }
    
        if (a == b && a == c){
            System.out.println("TRIANGULO EQUILATERO");
        }
        if ((a == b && a != c) || (b == c && b != a) || (a == c && a != b)){
            System.out.println("TRIANGULO ISOSCELES");
        }
    }
}